config/remote/firebase.go

```diff
@@ -63,56 +64,84 @@ func NewRConfig(updatePeriod time.Duration, rs RemoteConfigService, cm config.Ma
 func (rc *RConfig) fetchRemoteConfigIfTime() error {
 	var cfg config.Config
 	if err := rc.configManager.Load(&cfg); err != nil {
+		log.Println(internal.ErrorPrefix, "failed to load the cached config:", err)
```

Comment:
Should either log the error or return it, never both. If it needs to be logged, it should be logged by the caller then.